org.hydrogenmusic.hydrogen.json

```diff
@@ -0,0 +1,174 @@
+{
+  "app-id": "org.hydrogenmusic.hydrogen",
+  "branch": "stable",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
+  "sdk": "org.kde.Sdk",
+  "command": "hydrogen",
+  "rename-desktop-file": "hydrogen.desktop",
+  "rename-appdata-file": "hydrogen.appdata.xml",
```

Comment:
It would be nice if you could upstream those changes!

Reply:
see hydrogen-music/hydrogen#689